nodeup/pkg/model/kube_scheduler.go

```diff
@@ -192,7 +192,7 @@ func (b *KubeSchedulerBuilder) buildPod(kubeScheduler *kops.KubeSchedulerConfig)
 		flags = append(flags, "--"+flag+"kubeconfig="+defaultKubeConfig)
 	}
 
-	if kubeScheduler.UsePolicyConfigMap != nil {
+	if kubeScheduler.UsePolicyConfigMap != nil && *kubeScheduler.UsePolicyConfigMap {
```

Comment:
fi.BoolValue would be better in this case:
https://github.com/kubernetes/kops/blob/97869057129b30ea284c3ed1bdf1db36e752701d/pkg/values/values.go#L48-L53
```suggestion
	if fi.BoolValue(kubeScheduler.UsePolicyConfigMap) {
```

Reply:
ohh.. thats great! This is better.